spec/rails_helper.rb

```diff
@@ -66,6 +66,8 @@
   config.filter_rails_from_backtrace!
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
+
+  config.include Warden::Test::Helpers
```

Comment:
https://github.com/CircuitVerse/CircuitVerse/blob/bfe6a5884b65eca5b7de58ddd328dd39d4a13a73/spec/rails_helper.rb#L43 

Adding system here should work

Reply:
hmm it still doesn't work

Reply:
@tachyons